packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.html

```diff
@@ -238,12 +211,16 @@
                 </oui-tile-definition>
             </oui-tile>
         </div>
-        <div class="col-md-4">
+        <div
+            class="col-md"
+            data-ng-if="!$ctrl.isFeatureActivated('hideMiddleColumn')"
```

Comment:
No name more explicit than `MiddleColumn`?

Reply:
Middle column content changes with engines, I have not found a more explicit name. However, this feature will be removed later, when new features will appear on the middle column for each engine.